webapp/src/reducers/projectDetails/index.ts

```diff
@@ -1,24 +1,24 @@
 import {createSlice, PayloadAction} from '@reduxjs/toolkit';
 
 const initialState: ProjectDetails = {
-    id: '',
-    organization: '',
-    title: '',
+    projectID: '',
+    projectName: '',
+    organizationName: '',
 };
 
 export const projectDetailsSlice = createSlice({
     name: 'projectDetails',
     initialState,
     reducers: {
         setProjectDetails: (state: ProjectDetails, action: PayloadAction<ProjectDetails>) => {
-            state.id = action.payload.id;
-            state.title = action.payload.title;
-            state.organization = action.payload.organization;
+            state.projectID = action.payload.projectID;
+            state.projectName = action.payload.projectName;
+            state.organizationName = action.payload.organizationName;
         },
         resetProjectDetails: (state: ProjectDetails) => {
-            state.id = '';
-            state.title = '';
-            state.organization = '';
+            state.projectID = '';
+            state.projectName = '';
+            state.organizationName = '';
```

Comment:
Can't we do `state = initialState` here?

Reply:
no, and we should not do this even in the normal redux flow because object is passed by reference so after manipulation `initialState` does not have initial values i.e. empty data anymore